libs/ledger-live-common/src/deviceSDK/tasks/getDeviceInfo.ts

```diff
@@ -40,17 +39,7 @@ function internalGetDeviceInfoTask({
   return new Observable((subscriber) => {
     return (
       withDevice(deviceId)((transport) =>
-        getAppAndVersion(transport).pipe(
-          filter(({ appAndVersion: { name } }) => {
-            if (!isDashboardName(name)) {
-              subscriber.next({
-                type: "taskError",
-                error: "DeviceOnDashboardExpected",
-              });
-              return false;
-            }
-            return true;
-          }),
+        quitApp(transport).pipe(
```

Comment:
Smart - i guess we don't need a `getAppAndVersion` after all, if `quitApp` does nothing when the device is already on the dashboard ?

Reply:
I'll do some cleaning on `GetDeviceInfoTaskError` when i'll rework the genuine check - because we don't need it anymore, but we want to define a `taskError` maybe ?

Reply:
done in this PR actually